docs/scenarios_authoring/HandlingSchemas.md

```diff
@@ -0,0 +1,113 @@
+    * TODO: Support for [range](https://json-schema.org/understanding-json-schema/reference/numeric.html#range) and do proper conversion to: Float/Double/BigDecimal
+  * [String](https://json-schema.org/understanding-json-schema/reference/string.html#format) with [formats](https://json-schema.org/understanding-json-schema/reference/string.html#format):
+    * `date-time` => `java.time.ZonedDateTime`
+    * `date` => `java.time.LocalDate`
+    * `time` => `java.time.OffsetTime`
+* [Enum](https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values) => `java.lang.String`
+* `Object` => `java.util.HashMap`
+  * Static properties (fields) typing
+  * `additionalProperties` are supported, but additional fields won't be available during the hinting on the Designer. 
+  To get add field you have to do `#inpute.get("additional-field")`, but remember the result of this expression will be `Unknown`.
+  * Object without properties and `additionalProperties` is treated as `Map[String, Type]`, where `Type` is depends on
+  `additionalProperties` configuration: `true` means `Unknown`, `{"type": }` will be converted to a specific type.
+* Array
+* [Schema Composition](https://json-schema.org/understanding-json-schema/reference/combining.html)
+  * Nussknacker supports just `anyOf` and `oneOf` to build static typing information about the field. 
+
+### Sink validation & encoding
+
+Nussknacker doesn't support `object validation` (on the Designer level):
+* [Pattern properties](https://json-schema.org/understanding-json-schema/reference/object.html#pattern-properties)
+* [UnevaluatedProperties](https://json-schema.org/understanding-json-schema/reference/object.html#unevaluated-properties)
+* [Extending Closed Schemas](https://json-schema.org/understanding-json-schema/reference/object.html#extending-closed-schemas)
+* [Property names](https://json-schema.org/understanding-json-schema/reference/object.html#property-names)
+* [Size](https://json-schema.org/understanding-json-schema/reference/object.html#size)
+
+Nussknacker doesn't support `array validation` (on the Designer level):
+* [Additional Items](https://json-schema.org/understanding-json-schema/reference/array.html#additional-items)
+* [Tuple validation](https://json-schema.org/understanding-json-schema/reference/array.html#tuple-validation)
+* [Contains](https://json-schema.org/understanding-json-schema/reference/array.html#contains)
+* [Min/Max](https://json-schema.org/understanding-json-schema/reference/array.html#mincontains-maxcontains)
+* [Length](https://json-schema.org/understanding-json-schema/reference/array.html#length)
+* [Uniqueness](https://json-schema.org/understanding-json-schema/reference/array.html#uniqueness)
+
+These, configurations are supported by everit and on trying to serialize data will be blown up.
+
+Nussknacker supports only `anyOf` from `schema composition`. We decided that option is closest to union. Of course, you can use
```

Comment:
also `oneOf` ?

Reply:
Hmm, probably you're right: SwaggerTyped:60 ?